### DISCUSSION TITLE
How to enable image inversion for a given site?

### DISCUSSION THREAD

**Discussion by Qhilm** [darkreader/darkreader#13500] [Q&A] [answered]
Filter, Filter+ and Dynamic do NOT invert images for me. (Firefox 132)

I have a site where I do want images to be inverted, but I cannot figure how to do it – I'm sure I'm missing something trivial.

**Comment by Myshor** [comment]
Normally Dark Reader do not invert images in this modes. If you want to have them inverted then you need to INVERT them manually in sitefixes available under Dark Readers developer tools.

Other possibility is to create bug that some pictures are not visible/readable on the site then Dark Reader Team will try to make sitefix for that. 😉

**Reply by Qhilm** [reply]
Is there a way in dev tools to enable image inversion for all images on a specific site?

**Reply by Myshor** [reply, marked as answer]
if they are all presented as <img> then yoou can just add sitefix:

```
[pageURL]

INVERT
img

```

If there are some images used as backgrounds and/or in a different way (svg can be implemented in html totally implemented into the code) then you must include all selectors under INVERT section.

**Reply by Qhilm** [reply]
Thanks @Myshor, it worked, but I left the brackets out.

```
domain.com

INVERT
img
```

**Reply by Myshor** [reply]
Good. I use brackets if I use some name that needs to be changed. Just my style of showing something that must be changed with thing described in brackets. 😉
